lib/prompts/input.js

```diff
@@ -42,9 +42,14 @@ class InputPrompt extends Base {
     var bottomContent = '';
     var message = this.getQuestion();
     var transformer = this.opt.transformer;
+    var answerTransformer = this.opt.answerTransformer;
 
     if (this.status === 'answered') {
-      message += chalk.cyan(this.answer);
+      if (answerTransformer) {
+        message += answerTransformer(this.answer, this.answers);
```

Comment:
> What about a flag passed to the transformer function? `isFinal` or something like this (@SBoudrias)

We should transform answer in `answered` case, how can we send the `isFinal` flag to here?

Reply:
add a 3rd argument? `{ isFinal: this.status === 'answered' }`